cmd/bundle.rb

```diff
@@ -18,6 +18,8 @@ def bundle_args
 
         You can skip the installation of dependencies by adding space-separated values to one or more of the following environment variables: `HOMEBREW_BUNDLE_BREW_SKIP`, `HOMEBREW_BUNDLE_CASK_SKIP`, `HOMEBREW_BUNDLE_MAS_SKIP`, `HOMEBREW_BUNDLE_WHALEBREW_SKIP`, `HOMEBREW_BUNDLE_TAP_SKIP`.
 
+        You can force installation of a formula that does not have a bottle by setting the `HOMEBREW_INSTALL_IF_BOTTLE_MISSING` environment variable to `true`.
```

Comment:
```suggestion
        You can force the installation of a formula that does not have a bottle by setting the `HOMEBREW_BUNDLE_INSTALL_IF_BOTTLE_MISSING` environment variable.
```

Reply:
Agreed, if we do want an environment variable it should be a `HOMEBREW_BUNDLE` one.